nemo/collections/common/data/lhotse/nemo_adapters.py

```diff
@@ -429,7 +435,10 @@ def basename(d: dict) -> str:
                     del raw_audio
                     yield from cuts_for_recording
             except tarfile.ReadError:
-                logging.warning(f"Skipping tar file due to read errors (unstable storage or bad file?): {tar_path=}")
+                logging.warning(
+                    f"Skipping tar file due to read errors (unstable storage or bad file?): {tar_path=}",
+                    mode=logging_mode.ONCE,
```

Comment:
Please revert, if we have a lot of issues with data access, it should be clear from the logs.

Reply:
Looking at the implementation of `logging_mode.ONCE`:

```python
    def _logged_once(self, msg, mode):
        PREFIX_LEN = 12
        if mode == LogMode.ONCE:
            if msg[PREFIX_LEN:] in self.once_logged:
                return True
            self.once_logged.add(msg[PREFIX_LEN:])
        return False
```

It only considers the first 12 characters, in that case we will only identify the first corrupted file. If it was considering the full message, I would have been OK with that approach, but as-is it's not good enough.

Reply:
I see, just reverted the change